workflow.rst

```diff
@@ -248,7 +248,7 @@ machine type, use ``camelCased workflow name + StateMachine``::
     {
         private $blogPublishingWorkflow;
 
-        // this injects the blog_publishing workflow configured before
+        // Symfony will inject the blog_publishing workflow configured before
```

Comment:
To be honest, I don't see any difference ... and we never adds comments like this in the docs 😐